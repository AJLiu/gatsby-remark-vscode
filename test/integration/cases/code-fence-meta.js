module.exports = {
  wrapperClassName: 'test-wrapper',
  getWrapperClassName: ({ language, parsedOptions }) => {
    const { wrapperClass } = parsedOptions;
    if (wrapperClass) {
      return `${language}-${wrapperClass}`;
    }
  },
};
