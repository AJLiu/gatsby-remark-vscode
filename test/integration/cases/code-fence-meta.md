```js {1}
// should be highlighted
```

```js {wrapperClass: 'test'}
// should be highlighted
```