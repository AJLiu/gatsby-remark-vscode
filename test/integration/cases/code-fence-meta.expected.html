<pre class="grvsc-container test-wrapper default-dark" data-language="js"><code class="grvsc-code"><span class="grvsc-line grvsc-highlighted"><span class="mtk3">// should be highlighted</span></span></code></pre>
<pre class="grvsc-container test-wrapper js-test default-dark" data-language="js"><code class="grvsc-code"><span class="grvsc-line"><span class="mtk3">// should not be highlighted</span></span></code></pre>
<style class="grvsc-styles">
  :root {
    --vscode-highlight-padding-v: 1rem;
    --vscode-highlight-padding-h: 1.5rem;
    --vscode-highlight-padding-top: var(--vscode-highlight-padding-v);
    --vscode-highlight-padding-right: var(--vscode-highlight-padding-h);
    --vscode-highlight-padding-bottom: var(--vscode-highlight-padding-v);
    --vscode-highlight-padding-left: var(--vscode-highlight-padding-h);
    --vscode-highlight-border-radius: 8px;
  
    --vscode-highlight-line-highlighted-background-color: transparent;
    --vscode-highlight-line-highlighted-border-width: 4px;
    --vscode-highlight-line-highlighted-border-color: transparent;
  }
  
  .grvsc-container {
    overflow: auto;
    -webkit-overflow-scrolling: touch;
    padding-top: 1rem;
    padding-top: var(--vscode-highlight-padding-top);
    padding-bottom: 1rem;
    padding-bottom: var(--vscode-highlight-padding-bottom);
    border-radius: 8px;
    border-radius: var(--vscode-highlight-border-radius);
    font-feature-settings: normal;
  }
  
  .grvsc-code {
    display: inline-block;
    min-width: 100%;
  }
  
  .grvsc-line {
    display: inline-block;
    box-sizing: border-box;
    width: 100%;
    padding-left: 1.5rem;
    padding-left: var(--vscode-highlight-padding-left);
    padding-right: 1.5rem;
    padding-right: var(--vscode-highlight-padding-right);
  }
  
  .grvsc-highlighted {
    background-color: var(--vscode-highlight-line-highlighted-background-color);
    box-shadow: inset var(--vscode-highlight-line-highlighted-border-width) 0 0 0 var(--vscode-highlight-line-highlighted-border-color);
  }
  
  .default-dark {
    background-color: #1E1E1E;
    color: #D4D4D4;
  }
  .default-dark .mtk3 { color: #6A9955; }
</style>